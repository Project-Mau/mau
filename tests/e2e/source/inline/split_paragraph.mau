This is a paragraph split in two lines.
The missing empty line makes the two line belong to the same paragraph.
