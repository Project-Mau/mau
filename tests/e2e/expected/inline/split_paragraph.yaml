data:
  args: []
  content:
  - data:
      args: []
      content:
      - data:
          args: []
          kwargs: {}
          subtype: null
          tags: []
          type: text
          value: This is a paragraph split in two lines. The missing empty line makes
            the two line belong to the same paragraph.
      kwargs: {}
      subtype: null
      tags: []
      title: {}
      type: paragraph
  kwargs: {}
  subtype: null
  tags: []
  type: document

